### PR TITLE
Erkan

### PR DESCRIPTION
**Pull request by 1erkan1**
Local test passed, ready for review
Thank you

### CODE REVIEW

File: src/MyPractices/Test.java [hunk 1]
```diff
@@ -4,6 +4,6 @@ public class Test {
 
     public static void main(String[] args) {
         System.out.println("Hello World");
-        System.out.println("Im a professional Tester");
+        System.out.println("Test Updated");
```
Comment (Ozleblebici) [inline src/MyPractices/Test.java:7]: which test?

Comment (1erkan1) [reply src/MyPractices/Test.java:7]: hello

Comment (Ozleblebici) [reply src/MyPractices/Test.java:7]: hello hello, which test updated?

Comment (1erkan1) [reply src/MyPractices/Test.java:7]: Test class updated

Comment (Ozleblebici) [reply src/MyPractices/Test.java:7]: oh nice thats , I will approve it no need to change

**Review by Ozleblebici — APPROVED** [review]

**Review by Ozleblebici — COMMENTED** [review]
looks nice

**Review by Ozleblebici — APPROVED** [review]

**Pull request merged**